specs/gcc.anod

```diff
@@ -71,9 +80,28 @@ class GCC(spec('common')):
         args = ["--disable-nls", # Disable Native Language Support (NLS)
                 "--without-libiconv-prefix",
                 "--disable-libstdcxx-pch",
-                "--enable-lto",
-                "--with-gnu-as",
-                "--with-gnu-ld"]
+                "--enable-lto"]
+
+        if self.is_target_macos:
+            # This is what GNAT CE 2020 used to overcome the unusual
+            # arrangements in macOS Catalina and later.
+            args.append ("--with-build-config=no")
+            SDKROOT = \
+                subprocess.check_output (['xcrun', '--show-sdk-path'],
```

Comment:
I have to see if we can use the `self.shell()` function instead of `subprocess.check_output` here.